src/TabViewTypeDefinitions.js

```diff
@@ -58,4 +60,4 @@ export type PagerProps = {
   children?: *,
 };
 
-export type Style = Object | number | false | null | void | Array<Style>;
+export type Style = StyleObj;
```

Comment:
So this type `number | bool | Object | Array<?Atom>` seems incorrect because style doesn't actually accept a `boolean` (i.e. accepts `false`, but not `true`), but I guess we can't do anything here unless that's fixed. :(